### ISSUE TITLE
Graphic glitches when resizing split terminal

### ISSUE THREAD

**Issue by matthias-Q** [wez/wezterm#2483]
### What Operating System(s) are you seeing this problem on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

awesomewm 4.3

### WezTerm version

wezterm 20220904-064125-9a6cee2b

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

The issue occurs when I resize a horizontally splitted terminal. The prompt in the lower pane gets placed in a weird location.

When I unfocus them wezterm window, the prompt position gets updated. Therefor it is hard to get a screenshot of the problem.

### Configuration

```lua
local wezterm = require "wezterm"

local keys = {
    { key = "s", mods = "LEADER", action = wezterm.action { SplitVertical = { cwd = os.getenv "HOME" } } },
    { key = "v", mods = "LEADER", action = wezterm.action { SplitHorizontal = { cwd = os.getenv "HOME" } } },
    { key = "x", mods = "LEADER", action = wezterm.action { CloseCurrentPane = { confirm = false } } },
    { key = "c", mods = "LEADER", action = wezterm.action { SpawnCommandInNewTab = { cwd = os.getenv "HOME" } } },
    { key = "LeftArrow", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Left" } },
    { key = "RightArrow", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Right" } },
    { key = "UpArrow", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Up" } },
    { key = "DownArrow", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Down" } },
    { key = "h", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Left" } },
    { key = "l", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Right" } },
    { key = "k", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Up" } },
    { key = "j", mods = "ALT", action = wezterm.action { ActivatePaneDirection = "Down" } },
    { key = "0", mods = "CTRL", action = "ResetFontSize" },
    { key = "+", mods = "CTRL", action = "IncreaseFontSize" },
    { key = "-", mods = "CTRL", action = "DecreaseFontSize" },
    { key = "Tab", mods = "CTRL", action = { ActivateTabRelative = 1 } },
}

for i = 1, 8 do
    table.insert(keys, {
        key = tostring(i),
        mods = "LEADER",
        action = wezterm.action { ActivateTab = i - 1 },
    })
end

return {
    leader = { key = "a", mods = "CTRL", timeout_milliseconds = 1000 },
    keys = keys
}

```

### Expected Behavior

Prompt positions gets updated according to the size of the pane.

**Comment by jason0x43** [comment]
I believe I'm seeing the same thing with 20220904-212430-68380188 on macOS 12.5.1.

![2022-09-05 10 18 27](https://user-images.githubusercontent.com/1042866/188470170-70f0fb61-508f-40e5-a447-a4b84b7707a3.gif)

**Comment by wez** [comment]
This should be fixed now in `main`.

It typically takes about an hour before commits are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Comment by matthias-Q** [comment]
I built it from source and is fixed in 20220905-075210-6b81cd47 

Thanks.
